cfn.yaml

```diff
@@ -375,7 +375,7 @@
                                 {
                                     "Ref": "S3Key"
                                 },
-                                " app.tar.gz\ndocker load < app.tar.gz\ndocker run             --env-file .env             -p 3030:3030             --log-driver=awslogs             --log-opt awslogs-group=",
+                                " app.tar.gz\ndocker load < app.tar.gz\ndocker run --ulimit nofile=2048:2048 --env-file .env -p 3030:3030 --log-driver=awslogs --log-opt awslogs-group=",
```

Comment:
to be honest, given these are dedicated servers, you could go a lot higher.  The defult ulimit is assuming it's  shared server where one process is not supposed to take all the resources.
I think the ulimit for the membership/java AMI is baked in, off hand I think it's about 65536 or so.  You just want it to kick in before the OS dies.

Reply:
That's a lot bigger! I've increased it